src/components/TutorialPage/index.tsx

```diff
@@ -1,308 +1,285 @@
-                    <ul>
+                        </ul>
+                        <li>
+                            Once the file is loaded, you can copy the page URL
+                            and share this link with collaborators or post it on
+                            your website so that others can interactively view
+                            your results.
+                        </li>
+                    </ol>
+                </ul>
+            </ContentPagePanel>
+            <ContentPagePanel>
+                <ul className={styles.list}>
+                    <h3 className={styles.listHeader}>Browser support</h3>
+                    <ul className={styles.disc}>
                         <li>
                             Currently, Simularium supports Chrome, Safari, Edge,
                             and Firefox. Some features may not work on other
                             browsers.
                         </li>
                         <li>
-                            If using Safari on a Mac and the viewer is not
-                            working, update your browser to the most recent
-                            version. If you are using an older version, enable
-                            WebGL 2.0 by choosing Develop &gt; Experimental
-                            Features and enabling &quot;WebGL 2.0&quot; (If you
-                            do not have a Develop menu in your menu bar, first
-                            choose Safari &gt; Preferences &gt; Advanced and
-                            enable &quot;Show Develop menu in menu bar&quot;.)
-                            Then reload the viewer.
+                            If using Safari on a Mac, please enable WebGL 2.0 by
+                            choosing Develop &gt; Experimental Features and
+                            enabling &quot;WebGL 2.0&quot; (If you do not have a
+                            Develop menu in your menu bar, first choose Safari
+                            &gt; Preferences &gt; Advanced and enable &quot;Show
+                            Develop menu in menu bar&quot;.) Then reload the
+                            viewer.
                         </li>
                     </ul>
-                </div>
-            </Content>
-            <Footer />
-        </>
+                </ul>
+            </ContentPagePanel>
+
+            <ContentPagePanel isDark={true}>
+                <h3>Contact us</h3>
+                <p>
+                    We are collecting user feedback to improve this application.
+                    To find tutorials, seek technical support, report bugs or
+                    request features, use the Help menu found in the upper right
+                    corner of this website. Please contact us if you have ideas
+                    for potential collaborations, or want to be alerted to major
+                    updates.
+                </p>
+            </ContentPagePanel>
```

Comment:
I see two "Contact us" sections on this page:

![image](https://github.com/user-attachments/assets/b485cf62-9946-4186-81e4-d57040cf5044)